querydsl-core/src/main/java/com/querydsl/core/types/dsl/TimeExpression.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.querydsl.core.types.expr;
+package com.querydsl.core.types.dsl;
 
 import java.sql.Time;
 
```

Comment:
There are also instances of `@SuppressWarnings` [on the type level, like right under this comment](https://github.com/querydsl/querydsl/pull/1261/files#diff-0ef6d19b79529e691ba93255126408f7R32).
These are now not necessary anymore, and not desired.